src/Utilities/Compiler/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -40,6 +41,12 @@ internal static partial class AnalyzerOptionsExtensions
             CancellationToken cancellationToken)
             => options.GetNonFlagsEnumOptionValue(EditorConfigOptionNames.OutputKind, rule, s_defaultOutputKinds, cancellationToken);
 
+        public static EnumValuesPrefixTrigger GetEnumValuesPrefixTriggerOption(
+            this AnalyzerOptions options,
+            DiagnosticDescriptor rule,
+            CancellationToken cancellationToken)
```

Comment:
Might be good to match other API signatures in this file and have an additional parameter `EnumValuesPrefixTrigger defaultValue`, so each caller can decide it's default value.